src/xdocs/checks/javadoc/javadocblocktaglocation.xml

```diff
@@ -55,20 +56,16 @@
               <td>tags</td>
               <td>Specify the javadoc tags to process.</td>
               <td><a href="../../property_types.html#String.5B.5D">String[]</a></td>
-              <td><code>author, deprecated, exception, hidden, param, provides, return,
-                see, serial, serialData, serialField, since, throws, uses, version</code></td>
+              <td><code>author, deprecated, exception, hidden, param, provides, return, see, serial, serialData, serialField, since, throws, uses, version</code></td>
               <td>8.24</td>
             </tr>
             <tr>
               <td>violateExecutionOnNonTightHtml</td>
-              <td>
-                Control when to print violations if the Javadoc being examined by this check
-                violates the tight html rules defined at
-                <a href="../../writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.
-              </td>
+              <td>Control when to print violations if the Javadoc being examined by this check violates the tight html rules defined at <a href="../../writingjavadocchecks.html#Tight-HTML_rules">
+     Tight-HTML Rules</a>.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>false</code></td>
-              <td>8.24</td>
+              <td>8.3</td>
```

Comment:
We are not expecting a change in versions. Prove to me the original `8.24` is wrong.

Reply:
workaround-ed.
actual fix will be https://github.com/checkstyle/checkstyle/issues/14052

Reply:
If someone else is going to work on this, they we need to document with a comment that says "until ..."

Reply:
now, link is added at https://github.com/checkstyle/checkstyle/pull/14045/files#diff-582038065de9c242c07931401cfa262fcfd745638bf7c42736dba5793a3294cdR173
to avoid conflict, let not add it here also.